risc0/circuit/rv32im/src/prove/witgen/poseidon2.rs

```diff
@@ -191,13 +191,13 @@ impl Poseidon2 {
     pub fn read_node(ctx: &mut impl Risc0Context, node_idx: u32) -> Result<()> {
         // tracing::trace!("read_node: {node_idx:#010x}");
         let mut p2 = Poseidon2State::new_node(node_idx, true);
-        p2.rest(ctx, CycleState::PoseidonPaging)
+        p2.run(ctx, CycleState::PoseidonPaging)
```

Comment:
The name `rest` comes from the preflight implementation. There's an advantage to keeping the names similar so it's easier to ensure they remain consistent.